src/ActorMultiVertex.cpp

```diff
@@ -519,6 +519,7 @@ void ActorMultiVertex::UpdateAnimationState(bool force_update)
 					verts[i].t.x = rect.left;
 					verts[i].t.y = rect.bottom;
 					break;
+				// fall through
```

Comment:
I think just convert this to an if/else since it's looking for parity. No need to use a switch here imo.